rfd/0103-application-access-web-ui-auth-flow.md

```diff
@@ -1,74 +1,96 @@
 ---
-authors: Ryan Clark (ryan.clark@goteleport.com)
+authors: Ryan Clark (ryan.clark@goteleport.com), Lisa Kim (lisa@goteleport.com)
 state: implemented
 ---
 
-# RFD 103 - Application Access Web UI Auth Flow
+# RFD 103 - Application Access Web UI Authn Flow
 
 ## What
 
-This is an overview of the flow used for setting the relevant cookies when a user logs in to an
-application through Teleport Web UI.
+This is an overview of the authn flow used through the Teleport Web UI for setting the relevant cookies for when a user accesses an

```

Comment:
What is "authn flow", why is it called this way, and do I correctly understand that it's different from the regular flow with a proxy service?